src/tools/miri/src/eval.rs

```diff
@@ -61,7 +61,7 @@ pub enum IsolatedOp {
     /// Reject an op requiring communication with the host. By
     /// default, miri rejects the op with an abort. If not, it returns
     /// an error code, and prints a warning about it. Warning levels
-    /// are controlled by `RejectOpWith` enum.
+    /// are controlled by `RejectOpWith` en um.
```

Comment:
issue: oops?

Reply:
Ironic that the typo-fixing PR has a typo. This happened because I pressed the wrong button on the typo-fixing tool :sweat_smile: (Fixed)